src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -1144,17 +1143,25 @@ def _configure_security_policy_storage_params(arg_ctx):
                    help='The collation of the Azure SQL Managed Database collation to use, '
                    'e.g.: SQL_Latin1_General_CP1_CI_AS or Latin1_General_100_CS_AS_SC')
 
-    with self.argument_context('sql managed-db restore') as c:
+    with self.argument_context('sql midb restore') as c:
         create_args_for_complex_type(
             c, 'parameters', ManagedDatabase, [
                 'target_managed_database_name',
+                'target_managed_instance_name',
                 'restore_point_in_time'
             ])
+
         c.argument('target_managed_database_name',
                    options_list=['--target-db'],
```

Comment:
I just remembered that `sql db restore` uses `--dest-name` param names. See https://docs.microsoft.com/en-us/cli/azure/sql/db?view=azure-cli-latest#az-sql-db-restore Can you match that? Also `--dest-server`. `--target-db` and `--target-mi` sound good too, but I'd like to aim for consistency.